tofhir-engine/src/main/scala/io/tofhir/engine/cli/CommandFactory.scala

```diff
@@ -10,6 +10,8 @@ object CommandFactory {
       case "extract-redcap-schemas" => new ExtractRedCapSchemas()
       case "reload" => new Reload()
       case "run" | "execute" => new Run()
+      case "list" => new ListRunningMappings()
```

Comment:
When I type **help** (the command we use to list available commands), there is no l**ist** command